### ISSUE TITLE
`enforce-consistent-class-order` does not produce consistent results

### ISSUE THREAD

**Issue by wheesko** [schoero/eslint-plugin-better-tailwindcss#337]
### Description

Under the hood, the rule `enforce-consistent-class-order` uses `.localeCompare()` to sort the classNames list, preventing teams using different default system locales from having the same results. This is most obvious, when used with "asc" or "desc" order:

```js
'better-tailwindcss/enforce-consistent-class-order': [
  'error',
    {
      order: 'asc',
    },
],
```

Consider this code:
```
const testArray = ['px-12', 'py-24'];

console.log(testArray.toSorted((a, b) => a.localeCompare(b));
```

With `lt-LT` locale the output is:
``` ['py-24', 'px-12'] ```

With `en-US` the output is:
``` ['px-12', 'py-24'] ```

### Flavor

JavaScript

### Code Input

```tsx
-
```

### Expected Behavior

```tsx
The sorting is dependent on locale
```

### Actual Behavior

```tsx
The sorting should not be dependent on locale
```

### ESLint Log

```shell
-
```

### ESLint Config

```typescript
'better-tailwindcss/enforce-consistent-class-order': [
  'error',
    {
      order: 'asc',
    },
],
```

### Versions

- ESLint: 9.39.3
- Parser: -
- Plugin: 4.3.1
- Node: 22.14.0
